plugins/ruby-foreman-openscap/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: ruby-foreman-openscap
+Section: ruby
+Priority: extra
+Maintainer: Foreman Openscap Team
+Build-Depends: debhelper, foreman (>= 1.12.0~rc1), ruby-foreman-deface (<<2.0.0)
```

Comment:
If this works for 1.12, why should it not get cherry-picked to the 1.12 repos? :)